mysite/settings.py

```diff
@@ -115,3 +116,7 @@
 STATIC_URL = '/static/'
 
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+STATICFILES_DIRS = (
```

Comment:
Doesn't seem related either.

Reply:
I don't understand _why_ you did this. Also, why didn't you just assign what already exists?
```suggestion
STATICFILES_DIRS = STATIC_ROOT
```

Reply:
I dropped unrelated commits.